docs/writing-docs/tutorials.md

```diff
@@ -126,6 +126,17 @@ a ``"editor": "js"`` entry for JavaScript tutorials and ``"editor": "py"`` entry
 
 The tutorial markdown has a format that the guides the tutorial runner in making a sequence of interactions:
 
+### Metadata
+
+Tutorial metadata can be optionally specified in a table at the top of the document. The table is a key-value pairing of fields. The currrent fields are:
+
+* **v**: version, for the tutorial syntax. When unspecified, default is V1. Used in tutorial parsing.
+
+```markdown
```

Comment:
Does this table get hidden when rendering the tutorials as a docs page?

Reply:
Not right now, I can add that in the docs rendering update as well